Missing zoneinfo dependency on py3.8

kombu 5.3.0rc1, python 3.8, windows 11

```python
..\build_venv\lib\site-packages\celery\bin\worker.py:13: in <module>
    from celery.concurrency.base import BasePool
..\build_venv\lib\site-packages\celery\concurrency\base.py:13: in <module>
    from celery.utils import timer2
..\build_venv\lib\site-packages\celery\utils\timer2.py:14: in <module>
    from kombu.asynchronous.timer import Entry
..\build_venv\lib\site-packages\kombu\asynchronous\__init__.py:7: in <module>
    from .hub import Hub, get_event_loop, set_event_loop
..\build_venv\lib\site-packages\kombu\asynchronous\hub.py:19: in <module>
    from .timer import Timer
..\build_venv\lib\site-packages\kombu\asynchronous\timer.py:22: in <module>
    from backports.zoneinfo import ZoneInfo
E   ModuleNotFoundError: No module named 'backports'
```

The zoneinfo dependency was introduced by https://github.com/celery/kombu/pull/1680.

In requirements/test.txt we have `backports.zoneinfo>=0.2.1; python_version < '3.9'` so we should move that to defaults.txt

--- a/requirements/default.txt
+++ b/requirements/default.txt
@@ -1,3 +1,4 @@
 typing_extensions; python_version<"3.10"
 amqp>=5.1.1,<6.0.0
 vine
+backports.zoneinfo>=0.2.1; python_version < '3.9'

--- a/requirements/test.txt
+++ b/requirements/test.txt
@@ -3,4 +3,3 @@ pytest-sugar
 Pyro4
 hypothesis
 pytest-freezer
-backports.zoneinfo>=0.2.1; python_version < '3.9'

